Readme.md

```diff
@@ -84,44 +84,63 @@ the specifics for *pino-multi-stream*:
 
 [pinoapi]: https://github.com/pinojs/pino#api
 
-<a id="caveats"></a>
-## Caveats
+### pinoms.multistream(streams)
+
+Manually create a single `multistream` as used internally by the
+wrapper:
+
+```js
+var fs = require('fs')
+var pino = require('pino')
+var multistream = require('pino-multi-stream').multistream
+var streams = [
+  {stream: fs.createWriteStream('/tmp/info.stream.out')},
```

Comment:
so if you have level trace, and level info to separate streams

is it all logs from trace up to info that get sent to the trace stream?